gremlin-core/src/test/java/org/apache/tinkerpop/gremlin/language/translator/GremlinTranslatorTest.java

```diff
@@ -151,6 +151,15 @@ public static Collection<Object[]> data() {
                             null,
                             "g.with_(\"x\")",
                             "g.with_('x')"},
+                    {"g.inject(UUID(\"f47ac10b-58cc-4372-a567-0e02b2c3d479\"))",
+                            null,
+                            "g.inject(string0)",
+                            "g.Inject<object>(Guid.Parse(\"f47ac10b-58cc-4372-a567-0e02b2c3d479\"))",
+                            "g.Inject(uuid.Parse(\"f47ac10b-58cc-4372-a567-0e02b2c3d479\"))",
+                            null,
+                            null,
+                            "g.inject(\"f47ac10b-58cc-4372-a567-0e02b2c3d479\")",
```

Comment:
I don't think we want to translate to a simple string for JS. It should either use [utils.js](https://github.com/apache/tinkerpop/blob/5b7eea59a4264f06cc05a76383db2eda480379c4/gremlin-javascript/src/main/javascript/gremlin-javascript/lib/utils.js#L41-L44) (might need an extension there) or directly construct a uuid using the same uuid library.